staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -320,44 +320,53 @@ type PriorityLevelConfigurationList struct {
+	// that requests of this priority level are not subject to a limit
+	// (and thus are never queued) and do not detract from the
+	// capacity made available to other priority levels.  A value of
+	// `"Limited"` means that (a) requests of this priority level _are_
+	// subject to limits and might possibly be queued and (b) some of
+	// the server's limited capacity is made available exclusively to
+	// this priority level.
 	// Required.
-	Type PriorityLevelQueueingType `json:"type" protobuf:"varint,1,opt,name=type"`
+	// +unionDiscriminator
+	Type PriorityLevelEngagement `json:"type" protobuf:"bytes,1,opt,name=type"`
 
-	// `queuing` holds the configuration parameters that are
-	// only meaningful for a priority level that does queuing (i.e.,
-	// is not exempt).  This field must be non-empty if and only if
-	// `queuingType` is `"Queuing"`.
+	// `limited` specifies how requests are handled for a Limited priority level.
+	// This field must be non-empty if and only if `type` is `"Limited"`.
 	// +optional
-	Queuing *QueuingConfiguration `json:"queuing,omitempty" protobuf:"bytes,2,opt,name=queuing"`
+	Limited *LimitedPriorityLevelConfiguration `json:"limited,omitempty" protobuf:"bytes,2,opt,name=limited"`
 }
 
-// PriorityLevelQueueingType identifies the queuing nature of a priority level
-type PriorityLevelQueueingType string
+// PriorityLevelEngagement indicates whether a priority level is subject to limits on requests executing
+type PriorityLevelEngagement string
 
-// Supported queuing types.
+// Supported priority level engagement values.
 const (
-	// PriorityLevelQueuingTypeQueueing is the PriorityLevelQueueingType for priority levels that queue
-	PriorityLevelQueuingTypeQueueing PriorityLevelQueueingType = "Queuing"
+	// PriorityLevelEngagementExempt means that requests are not subject to limits
+	PriorityLevelEngagementExempt PriorityLevelEngagement = "Exempt"
 
-	// PriorityLevelQueuingTypeExempt is the PriorityLevelQueueingType for priority levels that are exempt from concurrency controls
-	PriorityLevelQueuingTypeExempt PriorityLevelQueueingType = "Exempt"
+	// PriorityLevelEngagementLimited means that requests are subject to limits
+	PriorityLevelEngagementLimited PriorityLevelEngagement = "Limited"
 )
 
-// QueuingConfiguration holds the configuration parameters that are specific to a priority level that is subject to concurrency controls
-type QueuingConfiguration struct {
-	// `assuredConcurrencyShares` (ACS) must be a positive number. The
-	// server's concurrency limit (SCL) is divided among the
-	// concurrency-controlled priority levels in proportion to their
-	// assured concurrency shares. This produces the assured
-	// concurrency value (ACV) for each such priority level:
+// LimitedPriorityLevelConfiguration specifies how to handle requests that are subject to limits.
+// It addresses two issues: what is the limit on request execution,
+// and what to do with requests that do not currently meet that limit.
```

Comment:
```
// It addresses two issues:
//  * How are requests for this PriorityLevel limited?
//  * What should be done with requests that exceed the limit?
```

Might be personal preference but I think this wording (if not the formatting) flows better.